airflow/gcp/hooks/gcs.py

```diff
@@ -357,7 +374,6 @@ def get_size(self, bucket_name, object_name):
     def get_crc32c(self, bucket_name, object_name):
         """
         Gets the CRC32c checksum of an object in Google Cloud Storage.
-
```

Comment:
This change prevents the documentation from being built properly. Can you delete it?